### DISCUSSION TITLE
[New Feature] Gain EXP when Healing/Repairing

### DISCUSSION THREAD

**Discussion by deathreaperz** [Phobos-developers/Phobos#975] [Ideas and feedback]
### Description:
Healing/Repairing an unit/infantry won't get Experience, thus it will be good if we can make it to get experience. (To get Veteran/EliteAbilities, EliteWeapons)

### Possible Implementation:
```ini
[FV]
Experience.FromEachHealing=10% (percentage, def 0%)
; If we can get exp based on the **each destroyed** target's cost, then the repair/heal exp will be obtained by **each healing** based on target's cost.
```

**Comment by amr-el-sayed** [comment]
I hope the same for the units like dogs, terror drones, mechanics, repair vehicles etc.
